README.md

```diff
@@ -1,2 +1,60 @@
-# op-component-library
+# Welcome to OP-Components library 👋
+
+![Version](https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/op-components#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
+
 One platform component library.
+
+## Install dependencies in all packages
+
+```sh
+lerna run install
+```
+
+## Usage
+
+### Development server
+
+- Run storybook
+
+```sh
+npm run dev
+```
+
+### Build all packages
+
+```sh
+lerna run build
+```
+
+## Run tests
+
+```sh
+npm run build
+npm run test
+```
+
+## Build an element using opc-generator
+
+### Installation
+
+First, install [Yeoman](http://yeoman.io) and generator-opc-generator using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
+```bash
+npm install -g yo
+npm install -g generator-opc-generator
+```
+
+Then generate your new project:
+
+```bash
+yo opc-generator
+```
+
+
+## 🤝 Contributors
```

Comment:
Add Anjnee, Ghanshayam and Me as contributors list too

Reply:
```

👤 **Ghanshyam Lohar** [@ghanlohar](https://github.com/ghanlohar)

👤 **Diwanshi Gadgil** [@diwanshi](https://github.com/diwanshi)

👤 **Anjnee K. Sharma** [@anjneeksharma](https://github.com/anjneeksharma)

👤 **Sumeet Ingole** [@gisumit](https://github.com/gisumit)

👤 **Deepesh Nair** [@hybridx](https://github.com/hybridx)
```
Added to the list